versioned_docs/version-0.4.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx

```diff
@@ -13,16 +13,16 @@ see [Prerequisites](/versioned_docs/version-0.4.0/getting-started-with-cloud-clu
 see [Preparing the clusters for registration](/versioned_docs/version-0.4.0/getting-started-with-cloud-clusters/prerequisites/preparing-the-cloud-clusters-for-registration.mdx).
 - You have installed Istio on the worker clusters to configure the external gateways. For more information, 
 see [Installing Istio](/versioned_docs/version-0.4.0/getting-started-with-cloud-clusters/installing-kubeslice/installing-the-kubeslice-controller.mdx#install-istio).
-- You have cluster administrator privileges to install the KubeSlice Controller on controller cluster and Slice Operator on worker clusters.
+- You have cluster administrator privileges to install the KubeSlice Controller on controller cluster and the Slice Operator on worker clusters.
 
 ## Install KubeSlice
 
-In this demonstration install KubeSlice on cloud clusters, using the custom topology configuration YAML.
+In this demonstration, install KubeSlice on cloud clusters using the custom topology configuration YAML.
 
 ## Install the KubeSlice Controller and Worker Clusters
 
-You must create a topology configuration file that includes the names of the clusters, the project name, and the cluster contexts
-that host the KubeSlice Controller and the worker clusters. For more information, see sample 
+You must create a topology configuration file that includes the names of the clusters and the cluster contexts that host the 
+KubeSlice Controller, the worker clusters, and a project name. For more information, see sample 
 [topology configuration](/versioned_docs/version-0.4.0/reference/sample-configuration-file.mdx#create-a-topology-configuration-file-for-cloud-clusters) file.
 
 Use the following command to install the controller and worker cluster:
```

Comment:
It is recommended to use article after the conjunction also 

install the controller and the worker clusters

Reply:
done